nncf/quantization/fake_quantize.py

```diff
@@ -355,11 +355,11 @@ def calculate_scale_zero_point(
     :return: Scale and Zero point values.
     """
     levels = level_high - level_low if narrow_range else level_high - level_low + 1
-    scale = ((input_high - input_low) / (levels - 1)).astype(TensorDataType.float32)
+    scale = ((input_high - input_low) * (1.0 / (levels - 1))).astype(TensorDataType.float32)
```

Comment:
I would suggest to run the conformance test for the post training quantization.

Reply:
Do you think this is neccessary? `calculate_scale_zero_point` is only used from `weight_lowering.py`.

Reply:
https://github.com/openvinotoolkit/nncf/blob/8d501c7951b6a6407775576b8e78959414223755/nncf/onnx/quantization/quantizer_parameters.py#L18